### ISSUE TITLE
AzureDataStudioPostgresExtension_Manage_Settings_DeveloperTool:The name for the format:Identifier case dropdown is defined as null.

### ISSUE THREAD

**Issue by nasc17** [microsoft/azuredatastudio#16622]
https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/1276722

ADS
Version: 1.32.0-insider (user setup)
Commit: 991b9af
Date: 2021-08-06T05:23:36.445Z
VS Code: 1.56.0
Electron: 12.0.7
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19043

Extension
PostgreSQL 0.2.7

**Repro Steps:**
Launch the application-Azure Data Studio and install extension PostgreSQL.
Tab till new connection button and press enter.
Tab till connection type dropdown and select postgresql.
Fill out the required details and tab till connect button and press enter.
Tab till manage button and press enter.
Tab till settings button and press enter.
Tab till search field and type pgsql.
Tab till format:Identifier case dropdown.
Open developer tool using ctrl+shift+I and check if the name is defined for dropdown.

**Actual:**
The name for the format:Identifier case dropdown is defined as null.

**Expected:**
The name should be defined as the label for the dropdown.

**User Impact:**
Screen Reader users will get wrong information regarding the combo box.

**Comment by alanrenmsft** [comment]
@nasc17 this is a vscode feature, please open an issue in VSCode repo and resolve your bug against it.
